x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/mock_endpoint_result_list.ts

```diff
@@ -73,14 +73,14 @@ export const mockHostDetailsApiResult = (): HostInfo => {
  * Mock API handlers used by the Endpoint Host list. It also sets up a list of
  * API handlers for Host details based on a list of Host results.
  */
-const hostListApiPathHandlerMocks = ({
-  hostsResults = mockHostResultList({ total: 3 }).hosts,
+const endpointListApiPathHandlerMocks = ({
+  endpointsResults: endpointResults = mockEndpointResultList({ total: 3 }).hosts,
```

Comment:
This does not look correct. Should be:

```suggestion
 endpointResults = mockEndpointResultList({ total: 3 }).hosts,
```

Reply:
:100: